addons/website_livechat/static/src/js/im_livechat.js

```diff
@@ -46,6 +46,26 @@ LivechatButton.include({
         }
     },
 
+    /**
```

Comment:
Now that I'm re-reading this, I think it could use a small docstring.

"When sending feedback, we want to blablabla because blablabla"